src/connector/src/sink/encoder/json.rs

```diff
@@ -705,14 +698,29 @@ mod tests {
                 ..mock_field.clone()
             },
             Some(ScalarRefImpl::Struct(StructRef::ValueRef { val: &value })),
-            DateHandlingMode::String,
-            TimestampHandlingMode::String,
-            TimestamptzHandlingMode::UtcString,
-            TimeHandlingMode::Milli,
-            &CustomJsonType::Doris(HashMap::default()),
+            &doris_config,
         )
         .unwrap();
         assert_eq!(interval_value, json!("{\"v3\":3,\"v2\":2,\"v1\":1}"));
+
+        let encode_jsonb_obj_config = JsonEncoderConfig {
+            time_handling_mode: TimeHandlingMode::Milli,
+            date_handling_mode: DateHandlingMode::String,
+            timestamp_handling_mode: TimestampHandlingMode::String,
+            timestamptz_handling_mode: TimestamptzHandlingMode::UtcString,
```

Comment:
Some test

Reply:
Can we move this refactor to its own PR?

Reply:
Sure

Reply:
stacked on #17706